docs/security/bot-protection.mdx

```diff
@@ -1,21 +1,20 @@
 ---
-title: Bot Protection
-description: Protect your sign-ups from bots
+description: Protect your sign-ups from bots.
 ---
 
-# Bot Protection
+# Bot protection
 
-To protect your Sign Ups from Bots, Clerk leverages data from our CDN to determine whether a user attempting to sign up might be a bot or not.
+To protect your sign-ups from bots, Clerk leverages data from our CDN to determine whether a user attempting to sign up might be a bot or not.
 
-Clerk offers two captcha widget display modes:
-1. **Smart**: The captcha widget will only be shown if the client is suspected to be a bot
-2. **Invisible**: No captcha widget will be displayed, even if the client is suspected to be a bot
+## Enable bot sign-up protection
 
-When set to **Smart**, users suspected of being a bot will be shown a interactive challenge (like clicking a checkbox), in an attempt to verify they are human. When set to **Invisible**, users suspected of being a bot will not be allowed to sign up, and are shown an error message instructing them to reach out to your configured support email address.
-
-To configure Bot Protection for your Sign Ups, head to the [Attack Protection](https://dashboard.clerk.com/last-active?path=user-authentication/attack-protection) section of the Clerk Dashboard.
+1. Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=user-authentication/attack-protection).
+1. Go to **User & Authentication > Attack Protection** in the sidebar menu.
+1. In the **Bot sign-up protection** section, enable the feature and choose the **CAPTCHA type** you want to use:
+    - **Smart**: The CAPTCHA widget will only be shown if the client is suspected to be a bot. Users suspected of being a bot will be shown a interactive challenge (like clicking a checkbox), in an attempt to verify they are human.
+    - **Invisible**: Bot traffic will be automatically blocked. No CAPTCHA widget will be displayed. Users suspected of being a bot will not be allowed to sign up, and are shown an error message instructing them to reach out to your configured support email address.
 
 ## Limitations
 
-- Bot Protection Captcha only supports web environments. Native environments are not supported yet, except if they are using a WebView.
-- If you are re-implementing Sign Ups with a [Custom Flow](/docs/custom-flows/overview) and have activated the **Smart** widget, then you need to ensure you have added a DOM node for the visible challenge. Refer to the [Custom Flow guide](/docs/custom-flows/bot-sign-up-protection) for more information.
+- Bot Protection CAPTCHA only supports web environments. Native environments are not supported yet, except if they are using a WebView.
```

Comment:
It might help to specify expo here rather than "native environments", since this is the only native environment we currently support. Also, the iOS sdk is actively working on adding bot protection, so it will likely be specific to expo by the time iOS SDK launches anyway

Reply:
resolved [6213012](https://github.com/clerk/clerk-docs/pull/1182/commits/62130125427a74ffc6be72a50c362c9c7746c411)